README.md

```diff
@@ -1 +1,68 @@
-# foundry-multichain-deploy
\ No newline at end of file
+# foundry-multichain-deploy
+
+Provides `foundry` tooling for the multichain deployment contract built atop of
+Sygma. See
+[ChainSafe/hardhat-plugin-multichain-deploy]("https://github.com/ChainSafe/hardhat-plugin-multichain-deploy")
+for more details.
+
+## Usage
```

Comment:
I think we are missing instructions to install this package into foundry first, no?